### ISSUE TITLE
 Invalid options object. Ignore Plugin has been initialized using an options object that does not match the API schema

### ISSUE THREAD

**Issue by Ajwah** [gatsbyjs/gatsby#33216]
### Preliminary Checks

- [X] This issue is not a duplicate. Before opening a new issue, please search existing issues: https://github.com/gatsbyjs/gatsby/issues
- [X] This issue is not a question, feature request, RFC, or anything other than a bug report directly related to Gatsby. Please post those things in GitHub Discussions: https://github.com/gatsbyjs/gatsby/discussions

### Description

```shell
 ERROR

Invalid options object. Ignore Plugin has been initialized using an options object that does not match the API schema.
 - options should be one of these:
   object { resourceRegExp, contextRegExp? } | object { checkResource }
   Details:
    * options misses the property 'resourceRegExp'. Should be:
      RegExp
      -> A RegExp to test the request against.
    * options misses the property 'checkResource'. Should be:
      function
      -> A filter function for resource and context.



  ValidationError: Invalid options object. Ignore Plugin has been initialized using an options object that does not match the API schema.
   - options should be one of these:
     object { resourceRegExp, contextRegExp? } | object { checkResource }
     Details:
      * options misses the property 'resourceRegExp'. Should be:
        RegExp
        -> A RegExp to test the request against.
      * options misses the property 'checkResource'. Should be:
        function
        -> A filter function for resource and context.

  - validate.js:105 validate
    [test]/[webpack]/[schema-utils]/dist/validate.js:105:11

  - create-schema-validation.js:16
    [test]/[webpack]/lib/util/create-schema-validation.js:16:17

  - IgnorePlugin.js:28 new IgnorePlugin
    [test]/[webpack]/lib/IgnorePlugin.js:28:3

  - webpack-plugins.ts:16 Object.ignore
    [test]/[gatsby]/src/utils/webpack-plugins.ts:16:12

  - webpack-utils.ts:768 Object.moment
    [test]/[gatsby]/src/utils/webpack-utils.ts:768:13

  - webpack.config.js:211 getPlugins
    [test]/[gatsby]/src/utils/webpack.config.js:211:15

  - webpack.config.js:512 module.exports
    [test]/[gatsby]/src/utils/webpack.config.js:512:14

  - build-html.ts:181 buildRenderer
    [test]/[gatsby]/src/commands/build-html.ts:181:18

  - start-server.ts:117 createIndexHtml
    [test]/[gatsby]/src/utils/start-server.ts:117:32

  - start-server.ts:155 startServer
    [test]/[gatsby]/src/utils/start-server.ts:155:5

  - start-webpack-server.ts:44 startWebpackServer
    [test]/[gatsby]/src/services/start-webpack-server.ts:44:7

```

### Reproduction Link

No need

### Steps to Reproduce

1. `npm init gatsby` (barebones installation. no selections)
2. `npm run develop`

### Expected Result

Opens browser local project

### Actual Result

Error

### Environment

```shell
System:
    OS: macOS 11.5.1
    CPU: (8) x64 Intel(R) Core(TM) i7-4770HQ CPU @ 2.20GHz
    Shell: 5.8 - /bin/zsh
  Binaries:
    Node: 16.6.1 - ~/.asdf/installs/nodejs/16.6.1/bin/node
    Yarn: 1.22.11 - /usr/local/bin/yarn
    npm: 7.20.3 - ~/.asdf/plugins/nodejs/shims/npm
  Languages:
    Python: 2.7.16 - /usr/bin/python
  Browsers:
    Chrome: 93.0.4577.82
    Firefox: 92.0
    Safari: 14.1.2
  npmPackages:
    gatsby: ^3.13.0 => 3.13.0
  npmGlobalPackages:
    gatsby: 3.13.0
```

### Config Flags

no. barebones project

**Comment by nooxx** [comment]
Same issue here

**Comment by nickloeper** [comment]
Webpack just released 5.53.0 which introduced this issue about an hour ago https://github.com/webpack/webpack/discussions/14265. I tried using resolutions although that didn't work either. This seems to be breaking all Gatsby 3 versions.

**Comment by barbalex** [comment]
So basically all versions of gatsby can not be used right now? 😱

**Comment by nickloeper** [comment]
This is only breaking for me on fresh installs since it is pulling the latest version of Webpack. If you have an existing lock file that has a previous version of webpack, you should be good until this is patched. If you are doing a fresh install, until this is fixed, you can use resolutions and choose a version < 5.53.0.

**Comment by barbalex** [comment]
```
  "resolutions": {
    "webpack": "5.52.1"
  },
```
works for me

**Comment by jreyes88** [comment]
Hi everyone, also getting this error this morning on a fresh GatsbyJS project.

**Comment by persigio** [comment]
Same error, after updating

**Comment by TylerAPfledderer** [comment]
Hey @barbalex! Are you using the [Force Resolutions Plugin](https://github.com/rogeriochaves/npm-force-resolutions) for the resolutions property or do you use Yarn or doing something else?

**Comment by barbalex** [comment]
I am using yarn

**Comment by alana314** [comment]
+1

**Comment by avrilpearl** [comment]
+1

**Comment by KyleAMathews** [comment]
This PR made previously optional parameters required https://github.com/webpack/webpack/pull/14211

**Comment by tdfirth** [comment]
+1 !

**Comment by Ayush-v** [comment]
same error.! don't know what to do. maybe this won't take too long to fix.

**Comment by Ayush-v** [comment]
> I am using yarn

not working either

**Comment by jonsherrard** [comment]
The resolutions field only works for nested dependencies if you're using Yarn. 

Unfortunately I cannot switch over to Yarn. 

I see there's a commit to pin webpack in Gatsby v3 in the meantime? Will that be installable soon?

**Comment by damnitrahul** [comment]
+1, Confirmed. Hopefully, this will be fixed soon.

**Comment by graysonhicks** [comment]
Fix PR is merged, waiting on release 👍

**Comment by jonsherrard** [comment]
Just out of interest @graysonhicks, which PR contains the fix? Learning opportunity for me to understand the nuts and bolts of Gatsby a bit better 😄

**Comment by snibbo71** [comment]
I rather wish I'd come here earlier, I might have some hair left if I had ;)

Will chill out now and wait for the new release - thanks graysonhicks

> Fix PR is merged, waiting on release 👍

**Comment by graysonhicks** [comment]
https://github.com/gatsbyjs/gatsby/pull/33222

Fixes the underlying webpack issue (I believe this was fixed a few days agowhen webpack made the change, and fixed on v4 branch but wasn't backported to v3).

**Comment by graysonhicks** [comment]
Fix published with 3.13.1

@pieh

**Comment by jonsherrard** [comment]
Works for me!

**Comment by snibbo71** [comment]
> Works for me!

Yep, looking good. That's absolutely fantastic service, thanks to everyone involved!

**Comment by pragmaticpat** [comment]
Confirmed as fixed, closing the issue - thank you @Ajwah for reporting!

**Comment by jonmadison** [comment]
thanks for the speedy fix! I've gotten past that issue and now have one with my remark markdown parser (needed for parsing the source code from my dev team) but for now, `npm install assert --save` works.

**Comment by krichey15** [comment]
> ```
>   "resolutions": {
>     "webpack": "5.52.1"
>   },
> ```
> 
> works for me

Where do you put this code?

**Comment by krichey15** [comment]
> Fix published with 3.13.1
> 
> @pieh

I am using Gatsby 3.2 and still experiencing this issue

**Comment by krichey15** [comment]
> Confirmed as fixed, closing the issue - thank you @Ajwah for reporting!

I am still experiencing this issue with 3.2

**Comment by 1-800-jono** [comment]
Using Gatsby v4.25.7 - just updated to webpack v5.61.0 (also webpack5.88.0) and seeing this error.
